components/dashboard/stats-card.tsx

```
@@ -40,46 +52,69 @@
+                id={`${title.toLowerCase().replace(/\s+/g, '-')}-title`}
+                className={TITLE_STYLES}
+              >
                 {title}
-              </p>
-              <p className="text-2xl font-bold text-gray-900 dark:text-gray-100">
-                {value}
+              </h3>
+              <p 
+                className={VALUE_STYLES}
+                aria-label={`${title}: ${formattedValue}`}
+              >
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)